boards/raspberry_pi_pico/Makefile

```diff
@@ -7,13 +7,11 @@ PLATFORM=raspberry_pi_pico
 include ../Makefile.common
 
 OPENOCD=openocd
-OPENOCD_INTERFACE=swd
-OPENOCD_OPTIONS=-f openocd-$(OPENOCD_INTERFACE).cfg
+OPENOCD_OPTIONS=-f openocd.cfg
```

Comment:
Revert this line to the original, as the Raspberry Pi can be used with both interfaces.